### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by pracsahu**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ appliesto:
...
 |Rolling out starting March 1, 2021<br> *(Complete)*  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|**GCC-High and DoD only**<br> Customers can now enable cloud meeting recordings in their Microsoft Teams for the first time. These recordings will be stored and played on OneDrive and SharePoint by default. |
+|Rolling out starting July 7, 2021<br> *(Complete)*  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|**All customers (Enterprise, Education, and GCC)**<br> For a Teams meeting that was recorded to OneDrive & SharePoint and was also transcribed live during the meeting you can now search in Microsoft Search to find the meeting recording file based on the transcript. |
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:33]:
```suggestion
|Rolling out starting July 7, 2021<br> *(Complete)*  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|**All customers (Enterprise, Education, and GCC)**<br> For a Teams meeting that was recorded to OneDrive and SharePoint and was also transcribed live during the meeting, you can now search in Microsoft Search to find the meeting recording file based on the transcript. |
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @pracsahu, kindly commit my suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**